template/tasks.py

```diff
@@ -207,6 +207,30 @@ def build_image(ctx, tag=None, pty=True):
     )
 
 
+@invoke.task
+def run_image(ctx, tag=None, options: str = None, command: str = None):
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏The typing is not quite correct. 

You cannot assign `None` to a parameter defined as a `str`. The type should be `str|None`.

Reply:
Updated [8a34b6d](https://github.com/ackama/django-template/pull/55/commits/8a34b6d670d6a5fbd0001c61d93e18dac94ef4e2)